components/resources/ai_chat_prompts.grdp

```diff
@@ -1,7 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <grit-part>
-  <message name="IDS_AI_CHAT_SUMMARIZE_PROMPT" translateable="false" desc="Default prompt for summarization">
-    Write a concise summary and always respond in bullet points:
-    <ph name="TEXT">$1</ph>
-  </message>
+  <message name="IDS_AI_CHAT_SUMMARIZE_PROMPT" translateable="false" desc="Default prompt for summarization">Here is an article in &lt;article&gt; tags:
+&lt;article&gt;
+<ph name="ARTICLE">$1</ph>
+&lt;/article&gt;
```

Comment:
Should we strip any string "</article>" and "<article>" form the article text?

Reply:
good point. We should probably do 👍